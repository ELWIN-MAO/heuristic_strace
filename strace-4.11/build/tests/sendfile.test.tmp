
mym_waitpid=102782

mym_waitpid=102782

mym_waitpid=102782

syscall_enter_mym_scno=59  sys_name=execve

mym_waitpid=102782

mym_waitpid=102782

syscall_exit_mym_scno=59  sys_name=execve

mym_waitpid=102782

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=102782

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=102782

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=102782

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=102782

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102782

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102782

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=102782

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=102782

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=102782

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=102782

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=102782

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=102782

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102782

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102782

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=102782

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=102782

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=102782

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=102782

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=102782

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=102782

syscall_enter_mym_scno=0  sys_name=read

mym_waitpid=102782

syscall_exit_mym_scno=0  sys_name=read

mym_waitpid=102782

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=102782

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=102782

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102782

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102782

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=102782

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=102782

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102782

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102782

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102782

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102782

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=102782

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=102782

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102782

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102782

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102782

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102782

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102782

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102782

syscall_enter_mym_scno=158  sys_name=arch_prctl

mym_waitpid=102782

syscall_exit_mym_scno=158  sys_name=arch_prctl

mym_waitpid=102782

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=102782

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=102782

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=102782

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=102782

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=102782

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=102782

syscall_enter_mym_scno=11  sys_name=munmap

mym_waitpid=102782

syscall_exit_mym_scno=11  sys_name=munmap

mym_waitpid=102782

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=102782

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=102782

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=102782

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=102782

syscall_enter_mym_scno=53  sys_name=socketpair

mym_waitpid=102782

syscall_exit_mym_scno=53  sys_name=socketpair

mym_waitpid=102782

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=102782

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=102782

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=102782

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=102782

syscall_enter_mym_scno=40  sys_name=sendfile
sendfile(0, 1, NULL, 4096
mym_waitpid=102782

syscall_exit_mym_scno=40  sys_name=sendfile

mym_scall_rval=0xffffffffffffffff
) = -1 EBADF (Bad file descriptor)

mym_waitpid=102782

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=102782

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=102782

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=102782

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=102782

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=102782

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=102782

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102782

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102782

syscall_enter_mym_scno=11  sys_name=munmap

mym_waitpid=102782

syscall_exit_mym_scno=11  sys_name=munmap

mym_waitpid=102782

syscall_enter_mym_scno=40  sys_name=sendfile
sendfile(0, 1, 0x7f3912a8f000, 4096
mym_waitpid=102782

syscall_exit_mym_scno=40  sys_name=sendfile

mym_scall_rval=0xffffffffffffffff
) = -1 EFAULT (Bad address)

mym_waitpid=102782

syscall_enter_mym_scno=40  sys_name=sendfile
sendfile(4, 5, NULL, 17979
mym_waitpid=102782

syscall_exit_mym_scno=40  sys_name=sendfile

mym_scall_rval=0x463b
) = 17979

mym_waitpid=102782

syscall_enter_mym_scno=40  sys_name=sendfile
sendfile(4, 5, 0x7f3912a8effc, 17979
mym_waitpid=102782

syscall_exit_mym_scno=40  sys_name=sendfile

mym_scall_rval=0xffffffffffffffff
) = -1 EFAULT (Bad address)

mym_waitpid=102782

syscall_enter_mym_scno=40  sys_name=sendfile
sendfile(4, 5, [0]
mym_waitpid=102782

syscall_exit_mym_scno=40  sys_name=sendfile
 => [17979], 17979
mym_scall_rval=0x463b
) = 17979

mym_waitpid=102782

syscall_enter_mym_scno=40  sys_name=sendfile
sendfile(4, 5, [17979]
mym_waitpid=102782

syscall_exit_mym_scno=40  sys_name=sendfile
 => [26968], 26969
mym_scall_rval=0x231d
) = 8989

mym_waitpid=102782

syscall_enter_mym_scno=40  sys_name=sendfile
sendfile(4, 5, [14627392582579060461]
mym_waitpid=102782

syscall_exit_mym_scno=40  sys_name=sendfile
, 1
mym_scall_rval=0xffffffffffffffff
) = -1 EINVAL (Invalid argument)

mym_waitpid=102782

syscall_enter_mym_scno=40  sys_name=sendfile
sendfile(4, 5, [233811181]
mym_waitpid=102782

syscall_exit_mym_scno=40  sys_name=sendfile
, 1
mym_scall_rval=0
) = 0

mym_waitpid=102782

syscall_enter_mym_scno=1  sys_name=write

mym_waitpid=102782

syscall_exit_mym_scno=1  sys_name=write

mym_waitpid=102782

syscall_enter_mym_scno=231  sys_name=exit_group

mym_waitpid=102782
+++ exited with 0 +++
