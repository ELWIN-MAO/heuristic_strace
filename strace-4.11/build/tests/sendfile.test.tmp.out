lo IP Address 127.0.0.1
ens33 IP Address 192.168.159.137
IP Address 127.0.0.1
IP Address 192.168.159.137
sendfile(0, 1, NULL, 4096) = -1 EBADF (Bad file descriptor)
sendfile(0, 1, 0x7f3912a8f000, 4096) = -1 EFAULT (Bad address)
sendfile(4, 5, NULL, 17979) = 17979
sendfile(4, 5, 0x7f3912a8effc, 17979) = -1 EFAULT (Bad address)
sendfile(4, 5, [0] => [17979], 17979) = 17979
sendfile(4, 5, [17979] => [26968], 26969) = 8989
sendfile(4, 5, [14627392582579060461], 1) = -1 EINVAL (Invalid argument)
sendfile(4, 5, [233811181], 1) = 0
+++ exited with 0 +++
